### PR TITLE
Add errbit errors so we can see the errors that occur

### PR DESCRIPTION
**Pull request by brentm5**
There were some 500 errors when looking at the gem so I decided to add this so we get notified.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -14,6 +14,7 @@ gem 'jquery-rails'
 gem 'turbolinks'
 gem 'jbuilder', '~> 1.0.1'
 gem 'clearance', '1.0.0.rc7'
+gem 'airbrake', '3.1.8'
```
Comment (drapergeek) [inline Gemfile:17]: Why are we locking the version here?

File: db/schema.rb [hunk 2]
```diff
@@ -13,9 +13,6 @@
 
 ActiveRecord::Schema.define(version: 20130511164059) do
 
-  # These are extensions that must be enabled in order to support this database
```
Comment (drapergeek) [inline db/schema.rb:16]: I don't want this change coming in and out on every PR. Remove it for now and we'll figure out what is going on.

File: Gemfile (hunk 1 above)
Comment (brentm5) [inline Gemfile:17]: Comparability with new airbrakr API / errbit

On Sunday, May 12, 2013, Jason Draper wrote:

> In Gemfile:
> 
> > @@ -14,6 +14,7 @@ gem 'jquery-rails'
> >  gem 'turbolinks'
> >  gem 'jbuilder', '~> 1.0.1'
> >  gem 'clearance', '1.0.0.rc7'
> > +gem 'airbrake', '3.1.8'
> 
> Why are we locking the version here?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/drapergeek/luckycharms/pull/35/files#r4186184
> .

## 

Brent Montague

Virginia Polytechnic Institute and State University

Computer Engineering

Bigbam505@gmail.com

Comment (drapergeek) [inline Gemfile:17]: 3.1.12 will not work? Patch level SHOULDN'T change anything.

Comment (brentm5) [inline Gemfile:17]: Does not work had it when I added it to tag-cloud etc.  It kept giving an odd error and when i looked it up people said 3.1.8 was the last version that worked correctly

**Pull request merged**